docs/DeveloperGuide.md

```diff
@@ -321,11 +325,186 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. System shows an error message prompting the user to put in a valid index.
 
       Use case resumes at step 2.
 
-*{More to be added}*
+* 3b. The user does not specify what type of details they want to change.
+
+    * 3b1. System shows an error message prompting the user to put in the type of details they want to edit.
+
+      Use case resumes at step 2.
+
+* 3c. The user does not specify what the new details should be.
+
+    * 3b1. System shows an error message prompting the user to put in the new details.
+
+      Use case resumes at step 2.
+
+* 3d. The user specifies details that do not meet the requirements of the detail type.
+
+    * 3b1. System shows an error message prompting the user with the correct detail type format and requirements.
+
+      Use case resumes at step 2.
+
+**Use case: UCSH02 Clear all contacts from the system**
```

Comment:
Is the clearing functionality here the same as UCSH04 (except that has an additional prompt)?

Reply:
Yes it is the same, just with the prompt - I wasn't sure if this is okay, but this is different functionality to just the clearing